### ISSUE TITLE
Design choices

### ISSUE THREAD

**Issue by gkasprow** [sinara-hw/RFSOC-AMC#2]
So far several aspects were decided:
- AMC form factor
- 2 AFE modules a'la Sayma
- RTM compatible with [RTM-SFP-QSFP](https://github.com/gkasprow/RTM_4SFP_3QSFP)
- AMC FatPipe1 support for DRTIO and PCIe
- clock racovery same as on Sayma (WR-type) + Xilinx devkit-style ADC/DAC clock generation
- 2x 64bit PL-DDR4
- 1x64bit PS-DDR4
- power supplies based on uModules like on Sayma RTM
- small FPGA for AFE control like on Sayma RTM

**Comment by tprzywoz** [comment]
@gkasprow  I've just published block schematics. Please check if connections on top schematic meet the requirements.

**Comment by drewrisinger** [comment]
Have there been any updates since March?

**Comment by gkasprow** [comment]
Yes, the schematics are completed. We also got funding so in January we will focus on routing.
